test/tests/problems/eigen_problem/eigensolvers/ne_coupled_scaled.i

```diff
@@ -87,12 +87,6 @@
     boundary = '0 1 2 3'
     value = 0
   [../]
-
-  [./eigenT]
-    type = EigenDirichletBC
-    variable = T
-    boundary = '0 1 2 3'
-  [../]
```

Comment:
I am puzzled why you deleted these? Could you explain a bit? Thanks.

Reply:
`T` is not an eigen vector in this problem, i.e. `c T` is not the solution like the fluxes. The purpose of `EigenDirichletBC` is to mark the variable it is on as an eigen variable. We do not see issues before because the old code only marks the eigen variables based on kernels, dg kernels and scalar kernels, but now we mark eigen variables based on all residual objects including nodal BC.

Reply:
I see. Good catch!